ios/19/_sentry_arm64.crash

```diff
@@ -0,0 +1,35 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BAD_ACCESS (SIGBUS)
+Crashed Thread: 0
+
+Application Specific Information:
+Attempted to dereference null pointer.
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x10004c2dc         -[CRLCrashOverwriteLinkRegister crash] (CRLCrashOverwriteLinkRegister.m:53)
+1   CrashLibiOS                     0x10004c2c0         -[CRLCrashOverwriteLinkRegister crash] (CRLCrashOverwriteLinkRegister.m:42)
```

Comment:
Reporting the same method twice with different lines and the method doesn't call itself recursively results in this being not complete. So this frame should be marked as invalid frame and the result for 64bit here is incomplete.